packages/react-reconciler/src/ReactFiberCommitHostEffects.js

```diff
@@ -517,6 +507,17 @@ function commitPlacement(finishedWork: Fiber): void {
     }
     parentFiber = parentFiber.return;
   }
+
+  if (!supportsMutation) {
+    if (enableFragmentRefs) {
+      appendImmutableNodeToFragmentInstances(
```

Comment:
I'm not sure what supportsMutation means, but wanted to point out that we are appending something in the fragment refs path which might be considered a mutation?